### PR TITLE
Fix app url to display invitation and make tests work

### PR DESCRIPTION
**Pull request by xmano**
3 changes addressed with this PR

1. Fix url to display invitation only for Cypress environment 
2. Skip create bucket test for now - will enable after implementing import
3. A new test in App financiers to get started.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -1595,7 +1595,7 @@
           "options": {
             "cypressConfig": "apps/financiers/financiers-e2e/cypress.json",
             "tsConfig": "apps/financiers/financiers-e2e/tsconfig.e2e.json",
-            "devServerTarget": "financiers:serve",
+            "baseUrl": "http://localhost:4200",
```
Comment (phpgeekfr) [inline angular.json:1598]: I'm not familiar with this, can you explain what does this change?

File: libs/invitation/src/lib/components/item/item.component.ts [hunk 2]
```diff
@@ -57,7 +57,15 @@ export class ItemComponent {
     private organizationService: OrganizationService,
     private userService: UserService,
     private routerQuery: RouterQuery
-  ) { }
+  ) { 
+    // @ts-ignore
+    if (window.Cypress) {
+      const host = location.protocol + '//' + location.hostname +
```
Comment (phpgeekfr) [inline libs/invitation/src/lib/components/item/item.component.ts:63]: in the project we usually do :
```
const host = `${location.protocol}//${location.hostname}${location.port ? ':' +location.port: }`
```

File: libs/invitation/src/lib/components/item/item.component.ts [hunk 3]
```diff
@@ -57,7 +57,15 @@ export class ItemComponent {
     private organizationService: OrganizationService,
     private userService: UserService,
     private routerQuery: RouterQuery
-  ) { }
+  ) { 
+    // @ts-ignore
+    if (window.Cypress) {
```
Comment (phpgeekfr) [inline libs/invitation/src/lib/components/item/item.component.ts:62]: can you add a comment explaining why this is here ?

File: libs/invitation/src/lib/components/item/item.component.ts [hunk 4]
```diff
@@ -57,7 +57,15 @@ export class ItemComponent {
...
+      const host = location.protocol + '//' + location.hostname + 
+                    (location.port ? ':' +location.port: '');
+      this.applicationUrl.festival = host;
```
Comment (phpgeekfr) [inline libs/invitation/src/lib/components/item/item.component.ts:65]: why is festival is hardcoded if we have `this.applicationUrl[this.app] = host;`  after ?

File: angular.json (hunk 1 above)
Comment (xmano) [reply angular.json:1598]: The default config of Nx has this but for our CI run it is launched already and we want to have just the baseUrl defined. It is in line with other apps.

File: libs/invitation/src/lib/components/item/item.component.ts (hunk 3 above)
Comment (xmano) [reply libs/invitation/src/lib/components/item/item.component.ts:62]: Ok

File: libs/invitation/src/lib/components/item/item.component.ts (hunk 4 above)
Comment (xmano) [reply libs/invitation/src/lib/components/item/item.component.ts:65]: Please refer to the function `eventLink()` I have just made it similar to it.

We need to change the URLs just for Cypress environment else tests are failing.

File: libs/invitation/src/lib/components/item/item.component.ts (hunk 2 above)
Comment (xmano) [reply libs/invitation/src/lib/components/item/item.component.ts:65]: I can change for the template literal

**Review by phpgeekfr — APPROVED** [review]

**Review by Mussuranae — APPROVED** [review]

**Pull request merged**